src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import duke.Task;
+import duke.Deadline;
+import duke.Todo;
+import duke.Event;
+import duke.DukeException;
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+
+    public static void main(String[] args) throws Exception {
+
+        ArrayList<Task> tasks = new ArrayList<>();
+        //scan for duke.txt file and add as input first
+        int i = 0;
+
+        File duke=new File("Duke.txt");
```

Comment:
Perhaps avoid the usage of magic literals. You can refactor this string literal into a constant (static final variable). [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)